src/com/zone5cloud/core/Z5ErrorItem.java

```diff
@@ -1,10 +1,27 @@
 package com.zone5cloud.core;
 
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
 public class Z5ErrorItem {
+	public static class Metadata {
+		@SerializedName(value = "required")
+		private List<String> requiredTerms;
+		
+		public List<String> getRequiredTerms() {
+			return this.requiredTerms;
+		}
+		
+		public void setRequiredTerms(List<String> terms) {
+			this.requiredTerms = terms;
+		}
+	}
 
 	private String field;
 	private Integer code;
 	private String message;
+	private Metadata metadata;
```

Comment:
I like this. Do we use "metadata" in other error objects? 

Some standards just use "meta" in the error response.